### PR TITLE
Supply some missing javadoc commentary

### PR DESCRIPTION
**Pull request by grumpyjames**
Work towards https://github.com/LMAX-Exchange/disruptor/issues/365

### CODE REVIEW

File: src/main/java/com/lmax/disruptor/EventProcessor.java [hunk 1]
```diff
@@ -38,5 +38,9 @@
      */
     void halt();
 
+    /**
+     * @return whether this event processor is running or not
+     * Implementations should ideally return true only when the associated thread is idle.
```
Comment (Palmr) [inline src/main/java/com/lmax/disruptor/EventProcessor.java:43]: > Implementations should ideally return true only when the associated thread is idle

I thought `isRunning` returns true only when the associated thread is _NOT_ idle?

Comment (grumpyjames) [reply src/main/java/com/lmax/disruptor/EventProcessor.java]: booleans will be booleans. Good catch, thank you.

File: src/main/java/com/lmax/disruptor/Sequenced.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 package com.lmax.disruptor;
 
+/**
+ * Operations related to the sequencing of items in a {@link RingBuffer}.
+ * See the two child interfaces, {@link Sequenced} and {@link EventSequencer} for more details.
```
Comment (Palmr) [inline src/main/java/com/lmax/disruptor/Sequenced.java:5]: Child interface would be Sequence**r**

**Review by Palmr — CHANGES_REQUESTED** [review]
Looks good, though think that link to `Sequenced` from `Sequenced` is incorrect.

**Review by Palmr — APPROVED** [review]

**Pull request merged**